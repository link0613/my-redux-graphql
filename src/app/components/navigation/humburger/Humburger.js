// @flow

// #region imports
import React from 'react';
// #endregion

// #region flow types
export type Props = { ...any };
// #endregion

const Humburger = () => (
  <button
    className="navbar-toggle collapsed"
    type="button"
    data-toggle="collapse"
    data-target="#bs-example-navbar-collapse-1"
  >
    <span className="sr-only">Toggle navigation</span>
    <span className="icon-bar" />
    <span className="icon-bar" />
    <span className="icon-bar" />
  </button>
);

// #region static props
Humburger.displayName = 'Humburger';
// #endregion

export default Humburger;
