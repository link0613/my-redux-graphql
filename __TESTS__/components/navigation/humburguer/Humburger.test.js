import React from 'react';
import Humburger from '../../../../src/app/components/navigation/humburger/Humburger';
import renderer from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('Humburger (navigation) component', () => {
  it('renders as expected', () => {
    const component = renderer.create(<Humburger />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
