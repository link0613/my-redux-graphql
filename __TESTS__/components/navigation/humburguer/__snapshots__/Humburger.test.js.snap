// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Humburger (navigation) component renders as expected 1`] = `
<button
  className="navbar-toggle collapsed"
  data-target="#bs-example-navbar-collapse-1"
  data-toggle="collapse"
  type="button"
>
  <span
    className="sr-only"
  >
    Toggle navigation
  </span>
  <span
    className="icon-bar"
  />
  <span
    className="icon-bar"
  />
  <span
    className="icon-bar"
  />
</button>
`;
